src/NLog.Extensions.Logging/Config/ExtensionLoggingConfiguration.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using NLog.Config;
+
+namespace NLog.Extensions.Logging
+{
+    /// <summary>
+    /// Configures NLog through Microsoft Extension Configuration section (Ex from appsettings.json)
+    /// </summary>
+    public class ExtensionLoggingConfiguration : LoggingConfigurationParser
```

Comment:
Please name it ...Parser

Proposal: SectionLoggingConfigurationParser

> ✓ CONSIDER ending the name of derived classes with the name of the base class.

from https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces

Reply:
Well `LoggingConfigurationParser` inherits from `LoggingConfiguration` that is the real base-class. It should not be called Parser.

The idea was that the name should match the existing `XmlLoggingConfiguration`. With the special touch that it was called `ExtensionLoggingConfiguration` that matches `ExtensionILoggerTarget`. But now this special touch has been lost because it now has to be called `MicrosoftILoggerTarget` and it would be confusing to name it `MicrosoftLoggingConfiguration`

Reply:
Maybe considering the name aspnetcore. Yes it's wrong, but Ms is also doing that sometimes, eg. https://github.com/aspnet/Extensions

Reply:
Renamed to NLogLoggingConfiguration